release/src/main.py

```diff
@@ -246,4 +249,24 @@ def notify_burgr(org,project,buildnumber,branch,sha1,status):
   url=f"{burgrx_url}/api/stage"
   r = requests.post(url, json=payload, auth=HTTPBasicAuth(burgrx_user, burgrx_password)) 
   if r.status_code != 201:          
-    print(f"burgr notification failed code:{r.status_code}" )   
\ No newline at end of file
+    print(f"burgr notification failed code:{r.status_code}" )   
+
+def check_public(project,buildnumber):
+  artifacts = get_artifacts_to_publish(project,buildnumber)
+  return "org.sonarsource" in artifacts
+
+def distribute_build(project,buildnumber):
+  payload={ 
+    "targetRepo": bintray_target_repo, 
+    "sourceRepos" : ["sonarsource-public-releases"]  
+  }
+  url=f"{artifactory_url}/api/build/distribute/{project}/{buildnumber}"
+  headers = {'content-type': 'application/json', 'X-JFrog-Art-Api': artifactory_apikey}
+  try:
+    r = requests.post(url, json=payload, headers=headers)  
+    r.raise_for_status()    
+    if r.status_code == 200:      
+      print(f"{project}#{buildnumber} pushed to bintray ready to sync to central")
+  except requests.exceptions.HTTPError as err:
+    print(f"Failed to distribute {project}#{buildnumber} {err}")
```

Comment:
It might be helpful in case of failure (exception or status != 200) to return the message "Failed to distribute {project}#{buildnumber}" in the HTTP response, in order to display a warning in the github action. WDYT ?